integration_tests/tests/consistency/consistency_account.sql

```diff
@@ -0,0 +1,32 @@
+{{ config(
+    tags="fivetran_validations",
+    enabled=var('fivetran_validation_tests_enabled', false)
+) }}
+
+-- this test is to make sure the final columns are the same between versions
+with prod as (
+    select *
+    from {{ ref('stg_salesforce__account') }}
+),
+
+dev as (
+    select *
+    from {{ ref('stg_salesforce__account') }}
+),
```

Comment:
These are both just pulling from the latest results being materialized. In this case it would be dev.

This would need to change to be the following in order for it to properly compare dev vs prod.
```suggestion
with prod as (
    select *
    from {{ target.schema }}_salesforce_source_prod.stg_salesforce__account
),

dev as (
    select *
    from {{ target.schema }}_salesforce_source_dev.stg_salesforce__account
),
```

Reply:
However, when I make this change locally I see quite a large failure in this test 🤔. Do you see the same failure after making this change?

Reply:
I imagine this same comment persists for all the other validations

Reply:
🤦‍♀️ I'll update them all! Thanks, @fivetran-joemarkiewicz As for failures, it makes sense since it would only work on data from the old connector since it's expected that the coalesce changes would bring in the renamed values instead of nulls on data from a new connector. 

I had considered how to make a better test and originally made tests that query the information schema to confirm the column lists and datatypes for each table from dev and prod were the same. I scrapped that since I would have to write it for only bigquery, but I am thinking maybe that is a better solution. Any other ideas?

Reply:
From our discussion at standup, I will write the test to query the information schema!